crates/epaint/src/image.rs

```diff
@@ -110,6 +110,15 @@ impl ColorImage {
         Self { size, pixels }
     }
 
+    /// Create a [`ColorImage`] from flat un-multiplied gray data.
+    ///
+    /// Panics if `size[0] * size[1] != gray.len()`.
+    pub fn from_gray_unmultiplied(size: [usize; 2], gray: &[u8]) -> Self {
```

Comment:
Since there is no alpha-channel here, the whole `unmultiplied` thing can be skipped:

```suggestion
    /// Create a [`ColorImage`] from flat opaque gray data.
    ///
    /// Panics if `size[0] * size[1] != gray.len()`.
    pub fn from_gray(size: [usize; 2], gray: &[u8]) -> Self {
```